aten/src/ATen/function_wrapper.py

```diff
@@ -119,6 +119,18 @@ def TypedDict(name, attrs, total=True):  # type: ignore
 .registerOp<${return_type} (${formals_types})>(Backend::${Backend}, "${schema_string}", &${Type}::${api_name})
```

Comment:
when I look at the generated code for this in TypeDefault I see that the backend here is Undefined?  Why?

Reply:
The TypeDefault functions are stored in the Undefined entry of the array, just because. No specific reason for it.